Composer/packages/client/src/pages/botProject/BotProjectInfo.tsx

```diff
@@ -24,14 +24,19 @@ const valueStyle = css`
 `;
 
 const headerStyle = css`
-  font-size: 16px;
+  font-size: 19px;
   font-weight: 600;
 `;
 
+const rootTextStyle = css`
+  color: gray;
+`;
+
 export const BotProjectInfo: React.FC<RouteComponentProps<{
   projectId: string;
+  isRootBot: boolean;
```

Comment:
Should this be optional if you're setting the default value to false?

Reply:
Handled